### PR TITLE
Allow specifying which subtree to pull

### PR DESCRIPTION
**Pull request by Veykril**

### CODE REVIEW

File: ferrocene/tools/pull-subtrees/pull.py [hunk 1]
```diff
@@ -385,6 +385,7 @@ def run_capture(*args, **kwargs):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--target", help="branch this pull is targeting", default=None)
+    parser.add_argument("--subtree", help="subtree to pull", default=None)
```
Comment (Dajamante) [inline ferrocene/tools/pull-subtrees/pull.py:388]: when will we use this flag?

**Review by Dajamante — APPROVED** [review]
bors merge

File: ferrocene/tools/pull-subtrees/pull.py (hunk 1 above)
Comment (Dajamante) [reply ferrocene/tools/pull-subtrees/pull.py:388]: @Veykril how will this be used?

Comment (Veykril) [reply ferrocene/tools/pull-subtrees/pull.py:388]: When you don't want to pull all subtrees at once

**Pull request merged**